examples/metrics/build.gradle

```diff
@@ -15,8 +15,13 @@
  */
 plugins {
 	id 'java'
+	id 'com.google.cloud.tools.jib' version '2.7.1'
 }
 
+// Set jib image path up using Google Container Registry
```

Comment:
You should call out this uses JIB in the README.md, and maybe a link to the gradle plugin docs: https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin.

Ideally this would have a default setting for folks consuming the example.  Maybe put one in here for now with the doc line you list so folks know to go and change it?

Something generic like:

```
jib.to.image = 'gcr.io/my-gcp-project/my-app'
```